<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="MainLayout" stretch="0,0,0">
      <item>
       <layout class="QVBoxLayout" name="LeftLayout" stretch="0">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="ParticipantListGroupBox">
          <property name="title">
           <string>Participant list</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QTableView" name="ParticipantTable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="ParticipantSearch">
             <property name="placeholderText">
              <string>Search participant...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="RightLayout" stretch="0,0">
        <item>
         <widget class="QGroupBox" name="ParticipantConstraintsGroupBox">
          <property name="title">
           <string>Participant constraints</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTreeView" name="ParticipantConstraintsTree">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="RoomGroupBox">
          <property name="title">
           <string>Room list</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QTableView" name="RoomTable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="RightLayout2">
        <item>
         <widget class="QGroupBox" name="SolutionGroupBox">
          <property name="title">
           <string>Solution</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTreeView" name="SolutionTree">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_from_file"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Rooms"/>
    <addaction name="actionLoad_participants"/>
    <addaction name="actionLoad_constraints"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_solution"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Rooms">
   <property name="text">
    <string>Load rooms</string>
   </property>
  </action>
  <action name="actionLoad_participants">
   <property name="text">
    <string>Load participants</string>
   </property>
  </action>
  <action name="actionLoad_constraints">
   <property name="text">
    <string>Load constraints</string>
   </property>
  </action>
  <action name="actionLoad_from_file">
   <property name="text">
    <string>Load all</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_solution">
   <property name="text">
    <string>Save solution</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
